business-wiki-frontend/shared/components/Layout.tsx

```diff
@@ -0,0 +1,18 @@
+import Head from "next/head";
+import Footer from "./Footer";
+
+const Layout = ({ children }) => {
```

Comment:
TS nie kazał Ci tu typować?

Reply:
Resolved in 1a901e0c61c41097d244d89f720d4500fa23577c